src/extern-sync/Tasks/SyncBoreholesTask.cs

```diff
@@ -53,6 +34,25 @@ protected override async Task RunTaskAsync(CancellationToken cancellationToken)
                 $"No suitable default user was found at target database." +
                 $"Was looking for a user with sub id <{targetDefaultUserSub}>.");
 
+        // Get published boreholes from the source database.
+        var publishedBoreholes = Source.BoreholesWithIncludes
+            .AsNoTrackingWithIdentityResolution()
+            .WithPublicationStatusPublished()
+            .ToList();
+
+        // Remove boreholes already available at the target database and workgroup by comparing
+        // the depth and coordinates of each borehole if they are within a pre-defined radius.
+        var boreholesAtDestination = await Target.Boreholes.AsNoTracking().Where(x => x.WorkgroupId == targetDefaultWorkgroup.Id).ToListAsync(cancellationToken).ConfigureAwait(false);
+        publishedBoreholes = [.. publishedBoreholes.RemoveDuplicates(boreholesAtDestination)];
```

Comment:
Q: Wenn ich den Code hier richtig interpretiere, erfolgt die Duplikatsprüfung neu nur noch gegen Bohrungen in der Default-/Fallback Workgroup. Eine Bohrung in der Workgroup 'foo' (Source & Destination) würde immer und immer wieder neu nach 'foo' kopiert. 🤔  Sollte der Vergleich nicht pro Bohrung im `foreach` Loop geschehen?